src/components/choice-tabs/ChoiceTabs.scss

```
@@ -130,7 +130,13 @@
   display: flex;
   align-items: baseline;
   gap: 7px;
-  flex-wrap: nowrap;
+  // Allow the title row to wrap so long labels never get clipped past the
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)